src/reducers/__tests__/query.spec.jsx

```diff
@@ -28,15 +28,16 @@ describe( 'reducer:query', () => {
         lens: 'Overview',
         mapWarningEnabled: true,
         queryString: '?date_received_max=2020-05-05' +
-          '&date_received_min=2017-05-05&field=all',
+          '&date_received_min=2017-05-05&field=all' + 
+          '&lens=overview&trend_depth=5&trend_interval=month',
         searchText: '',
         searchField: 'all',
         page: 1,
         searchAfter: '',
         size: 25,
         sort: 'created_date_desc',
         subLens: '',
-        tab: types.MODE_MAP,
+        tab: 'Trends',
```

Comment:
i think we have [types.MODE_TRENDS](https://github.com/cfpb/ccdb5-ui/blob/main/src/constants/index.jsx#L15)

Reply:
Hmm am I missing how this particular test is set up. I noticed in tests like https://github.com/cfpb/ccdb5-ui/blob/main/src/components/__tests__/RowChart.spec.jsx#L426 the literal value is used. Some schools of thought would say not to test against variable from the codebase in the test, because if the variable erroneously changed, the test would still pass. Which way makes sense here?

Reply:
Another example is `Trends` here --> https://github.com/cfpb/ccdb5-ui/blob/main/src/reducers/__tests__/query.spec.jsx#L384, while types.MODE_MAP is used here --> https://github.com/cfpb/ccdb5-ui/blob/main/src/reducers/__tests__/query.spec.jsx#L399

Reply:
I think the latter.   Doesn't really matter I think.  The values are pretty tightly coupled, but you're right, we should be consistent about how we do it.